{"name": "用户注册登录删除-预期失败", "status": "skipped", "statusDetails": {"message": "AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nassert False is True\n +  where False = <core.result_base.ResultBase object at 0x1038144a8>.success", "trace": "self = <testcases.scenario_test.test_03_register_login_delete.TestRegLogDelete object at 0x103814cc0>\ntestcase_data = {'address': '深圳市宝安区', 'admin_pwd': '123456', 'admin_user': 'wintest4', 'except_code': 3004, ...}\n\n    @allure.story(\"用例--注册/登录/删除--预期失败\")\n    @allure.description(\"该用例是针对 注册-登录-删除 场景的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"用户注册登录删除-预期失败\")\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason=\"xfail示例：预期失败的用例，如存在尚未解决的Bug等\")\n    @pytest.mark.usefixtures(\"delete_register_user\")\n    def test_user_register_login_delete_fail(self, testcase_data):\n        username = testcase_data[\"username\"]\n        password = testcase_data[\"password\"]\n        telephone = testcase_data[\"telephone\"]\n        sex = testcase_data[\"sex\"]\n        address = testcase_data[\"address\"]\n        admin_user = testcase_data[\"admin_user\"]\n        admin_pwd = testcase_data[\"admin_pwd\"]\n        except_result = testcase_data[\"except_result\"]\n        except_code = testcase_data[\"except_code\"]\n        except_msg = testcase_data[\"except_msg\"]\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register_user(username, password, telephone, sex, address)\n        step_1(username, password, telephone, sex, address)\n>       assert result.success is True, result.error\nE       AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nE       assert False is True\nE        +  where False = <core.result_base.ResultBase object at 0x1038144a8>.success\n\ntestcases/scenario_test/test_03_register_login_delete.py:87: AssertionError"}, "description": "该用例是针对 注册-登录-删除 场景的测试", "steps": [{"name": "步骤1 ==>> 注册用户", "status": "passed", "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "password", "value": "'123456'"}, {"name": "telephone", "value": "'13599999999'"}, {"name": "sex", "value": "'1'"}, {"name": "address", "value": "'深圳市宝安区'"}], "start": 1632654816186, "stop": 1632654816187}], "attachments": [{"name": "log", "source": "cd243730-d878-4008-9965-aca179a3314c-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0d7daf51-333b-443c-bd54-562bbb15d14c-attachment.txt", "type": "text/plain"}], "start": 1632654816173, "stop": 1632654816189, "uuid": "f588d2fd-5d65-43f0-950f-64cc7941b698", "historyId": "7164a98ce91bd9f479af2fa0b0f95820", "testCaseId": "b7d06749c352697ac0c07e2d04f5e9d2", "fullName": "testcases.scenario_test.test_03_register_login_delete.TestRegLogDelete#test_user_register_login_delete_fail", "labels": [{"name": "severity", "value": "minor"}, {"name": "story", "value": "用例--注册/登录/删除--预期失败"}, {"name": "epic", "value": "针对业务场景的测试"}, {"name": "feature", "value": "场景：用户注册-用户登录-删除用户"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "testcases.scenario_test"}, {"name": "suite", "value": "test_03_register_login_delete"}, {"name": "subSuite", "value": "TestRegLogDelete"}, {"name": "host", "value": "txxdembp"}, {"name": "thread", "value": "11611-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.scenario_test.test_03_register_login_delete"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}